rxandroid/src/main/java/rx/android/schedulers/AndroidSchedulers.java

```diff
@@ -18,27 +18,16 @@
 import android.os.Looper;
 import rx.android.plugins.RxAndroidPlugins;
 
-/**
- * Schedulers that have Android-specific functionality
- */
+/** Android-specific Schedulers. */
 public final class AndroidSchedulers {
     private AndroidSchedulers() {
         throw new AssertionError("No instances");
     }
 
     private static final Scheduler MAIN_THREAD_SCHEDULER =
-            new HandlerThreadScheduler(new Handler(Looper.getMainLooper()));
-
-    /**
-     * {@link Scheduler} which uses the provided {@link Handler} to execute actions.
-     */
-    public static Scheduler handlerThread(final Handler handler) {
-        return new HandlerThreadScheduler(handler);
-    }
+            new HandlerScheduler(new Handler(Looper.getMainLooper()));
 
-    /**
-     * {@link Scheduler} which will execute actions on the Android UI thread.
-     */
+    /** A {@link Scheduler} which executes actions on the Android UI thread. */
     public static Scheduler mainThread() {
         Scheduler scheduler =
                 RxAndroidPlugins.getInstance().getSchedulersHook().getMainThreadScheduler();
```

Comment:
I disagree that only 1% of library users will use the hook. Anyone who wants to do testing with RxAndroid will want this hook.